internal/tofu/context_apply.go

```diff
@@ -46,10 +46,31 @@ func (c *Context) Apply(plan *plans.Plan, config *configs.Config) (*states.State
 		// like to show some helpful output that mirrors the way we show other changes.
 		if rc.Importing != nil {
 			for _, h := range c.hooks {
-				// In future, we may need to call PostApplyImport separately elsewhere in the apply
+				// In the future, we may need to call PostApplyImport separately elsewhere in the apply
 				// operation. For now, though, we'll call Pre and Post hooks together.
-				h.PreApplyImport(rc.Addr, *rc.Importing)
-				h.PostApplyImport(rc.Addr, *rc.Importing)
+				_, err := h.PreApplyImport(rc.Addr, *rc.Importing)
+				if err != nil {
+					panic(err)
+				}
+				_, err = h.PostApplyImport(rc.Addr, *rc.Importing)
+				if err != nil {
+					panic(err)
+				}
+
+			}
+		}
```

Comment:
Hmm, I wonder why these errors were not caught in the first place? It feels like an oversight but there may have been a reason that it didn't result in a panic before.